web/modules/custom/geolayer/modules/map/config/schema/farm_map.schema.yml

```diff
@@ -0,0 +1,72 @@
+# Schema for the configuration files of the farm_map module.
```

Comment:
Make sure you run the reports of https://www.drupal.org/project/config_inspector on this